src/rdmnet/broker/linux/linux_socket_manager.cpp

```diff
@@ -95,7 +95,10 @@ bool LinuxBrokerSocketManager::Shutdown()
 
   etcpal::MutexGuard socket_guard(socket_lock_);
   for (auto& sock_data : sockets_)
+  {
+    shutdown(sock_data.second->socket, SHUT_RDWR);
```

Comment:
An excellent fix!  It's always a good idea to call shutdown() before close().  And easy to forget.

Reply:
Yeah, it was added because without it, the broker was not flushing its RDMnet disconnect message before disconnecting a client.